packages/math-input/src/components/keypad/__tests__/keypad-v2-mathquill.test.tsx

```diff
@@ -0,0 +1,185 @@
+import Color from "@khanacademy/wonder-blocks-color";
+import {Popover} from "@khanacademy/wonder-blocks-popover";
+import {render, screen} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import MathQuill from "mathquill";
+import * as React from "react";
+
+import "@testing-library/jest-dom";
+
+import Key from "../../../data/keys";
+import keyTranslator from "../../key-translator";
+import Keypad from "../index";
+
+type Props = {
+    onChangeMathInput: (mathInputTex: string) => void;
+};
+
+function V2KeypadWithMathquill(props: Props) {
+    const mathquillWrapperRef = React.useRef<HTMLDivElement>(null);
+    const [mathField, setMathField] = React.useState();
+
+    React.useEffect(() => {
+        if (!mathField && mathquillWrapperRef.current) {
+            const MQ = MathQuill.getInterface(2);
+            const mathFieldInstance = MQ.MathField(
+                mathquillWrapperRef.current,
+                {
+                    charsThatBreakOutOfSupSub: "+-*/=<>≠≤≥",
+                    handlers: {
+                        edit: () =>
+                            props.onChangeMathInput(mathFieldInstance.latex()),
+                    },
+                },
+            );
+            setMathField(mathFieldInstance);
+        }
+    }, [mathField, props]);
+
+    function handleClickKey(key: Key) {
+        if (!mathField) {
+            return;
+        }
+
+        const mathFieldCallback = keyTranslator[key];
+        if (mathFieldCallback) {
+            mathFieldCallback(mathField, key);
+        }
+    }
+
+    return (
+        <div style={{maxWidth: "400px", margin: "2em"}}>
```

Comment:
We don't need style on this line or 72-76. (RTL doesn't layout the components and we don't need to test the way they look.)